tutorials/06_Advanced_ChangeInputs.md

```diff
@@ -1,68 +1,77 @@
+The regional resolution of the run is set by 
+```R
 cfg$regionmapping
 ```
-(default setting: regionmapping <- "config/regionmappingH12.csv"). Based on the regional resolution and the input data revision 
-``` bash
+(default setting: `regionmapping <- "config/regionmappingH12.csv"`).
+Based on the regional resolution and the input data revision 
+```R
 cfg$inputRevision
 ```
-the name of the needed input data is constructed. It is checked whether those input data are already available. If not they are automatically downloaded from `/p/projects/rd3mod/inputdata/output/` and distributed.
-
-The prepared input data is a compressed tar archive file "`.tgz`", which can be opened with software such as [7-Zip](https://www.7-zip.org/), or in terminal by `tar` and `untar` commands. 
+the name of the needed input data is constructed. It is checked whether those input data are already available. If not they are automatically downloaded and distributed.
 
-Local Input Data
-==================
-If you like to get the input data on your local machine you need to copy your ssh key (`/home/[your_cluster_user_name]/.ssh/id_dsa`) to your computer and adjust your local **.Rprofile** by adding the following line:
-``` r
-options(remind_repos=list("scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output"=list(username="[your_cluster_user_name]",ssh_private_keyfile="[path_to_your_local_copy_of_your_key]")))
-```
-If your key is password-protected you have to remove this by using an empty passphrase:  
-``` cmd 
-ssh-keygen -p
--> path to id_dsa
--> empty passphrase
-```
-It is recommended to first copy the existing key (e.g. id_dsa into id_dsa-local) and remove the passphrase of this copy and copy it to your machine.
+The prepared input data is a compressed tar archive file `.tgz` and can be found on the cluster at `/p/projects/rd3mod/inputdata/output`.
+If you want to peek inside the archive to debug something or out of curiosity you can use the software [7-Zip](https://www.7-zip.org/), or the `tar` command in the terminal.
 
-If you plan to run REMIND not with the default regional resolution you have to take care that REMIND starts from a gdx with the correct regional resolution. Either you can use one from an older run with the corresponding regional resolution or you can construct a new gdx with the correct regional resolution from a gdx in a different regional resolution by using the function `gdx_rename` from the package (gdx) (e.g. `gdx_rename("input.gdx",set_name="all_regi",c(REF="RUS",CAZ="ROW",...,MEA="MEA",USA="USA"))`).
+### How to Update Input Data
 
+1. Check for the current input data revision number in this cluster folder: `/p/projects/rd3mod/inputdata/cache`. Alternatively, run the helper tool `lastrev` (`/p/projects/rd3mod/tools/lastrev`) to get a list of the last five revX.XXX*_remind.tgz items in the default madrat output directory.
```

Comment:
that is not correct any more, you need to check for the latest revision in the output folder (/p/projects/rd3mod/inputdata/output)

Reply:
jep